src/mbgl/shader/icon_shader.cpp

```diff
@@ -21,8 +21,8 @@ void IconShader::bind(GLbyte* offset) {
     MBGL_CHECK_ERROR(glEnableVertexAttribArray(a_offset));
     MBGL_CHECK_ERROR(glVertexAttribPointer(a_offset, 2, GL_SHORT, false, stride, offset + 4));
 
-    MBGL_CHECK_ERROR(glEnableVertexAttribArray(a_data1));
-    MBGL_CHECK_ERROR(glVertexAttribPointer(a_data1, 4, GL_UNSIGNED_BYTE, false, stride, offset + 8));
+    MBGL_CHECK_ERROR(glEnableVertexAttribArray(a_texture_pos));
+    MBGL_CHECK_ERROR(glVertexAttribPointer(a_texture_pos, 2, GL_UNSIGNED_SHORT, false, stride, offset + 8));
```

Comment:
Do we need to define `a_texture_pos` in `shader.hpp`? https://github.com/mapbox/mapbox-gl-native/blob/69c529e26463303dea81a0b5d4b8540a9384d482/src/mbgl/shader/shader.hpp